ibis/tests/all/test_join.py

```diff
@@ -45,18 +41,22 @@ def right_df(right):
         ),
     ],
 )
-def test_join_project_left_table(
-    backend, con, left, right, left_df, right_df, how
-):
+@pytest.mark.only_on_backends(all_db_join_supported)
+# Csv is a subclass of Pandas so need to skip it explicited
```

Comment:
There seems to a bug in the Csv backend that I couldn't quite make this test work, so I disabled it here.